config.js

```diff
@@ -53,30 +52,27 @@ config = Object.assign({}, config, {
   // ----------------------------------
   // Compiler Configuration
   // ----------------------------------
-  compiler_devtool: __DEV__ && 'cheap-source-map'
-  || __TEST__ && 'cheap-source-map'
-  || __STAGING__ && 'source-map',
+  compiler_devtool: __DEV__ && 'cheap-source-map' || __TEST__ && 'inline-source-map' || __STAGING__ && 'source-map',
+  compiler_globals: {
+    process: {
+      env: {
+        NODE_ENV: JSON.stringify(env),
+      },
+    },
+    __DEV__,
+    __DEBUG__: !!argv.debug,
```

Comment:
This is also cruft, note, this build system was copied from one of our production apps and not thoroughly updated, so it has some extra junk that we don't need.